product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_from_azure/index.mdx

```diff
@@ -53,6 +53,49 @@ PostgresAuditLogs_CL
 | project record_log_time_s, record_error_severity_s, record_message_s
 | sort by record_log_time_s desc
 ```
+ 
+### Using BigAnimal's cloud account
+
+To access your Postgres cluster logs, when using BigAnimal's cloud account, generate a SAS token from BigAnimal and use it to download the logs. 
+
+1. In the BigAnimal portal, select **Clusters**, select your cluster, and select the **Monitoring & Logging** tab. 
+
+1. Select **Generate Token** and copy the SAS token. The SAS token is a sensitive value and shouldn't be made publicly available. The following is a sample SAS token: 
+
+   ```
+   https://blobsamples.blob.core.windows.net/?sv=2022-11-02&ss=b&srt=sco&sp=rwlc&se=2023-05-24T09:51:36Z&st=2023-05-24T01:51:36Z&spr=https&sig=<signature> 
+   ```
+    
+1. Enter the `azcopy` command to download the Postgres logs from BigAnimal. For example:
+
+   ```
+   azcopy copy '$TOKEN' . --recursive
+   INFO: Scanning...
+   INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support
+   
+   Job aa4b74a0-bc92-be4e-551c-47aec1c1cfc3 has started
+   Log file is located at: /Users/valeriodelsarto/.azcopy/aa4b74a0-bc92-be4e-551c-47aec1c1cfc3.log
```

Comment:
should we put a fake name instead of mine here? Talking about `valeriodelsarto`

Reply:
Good idea. Made the change to `sample_user`.